src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -156,6 +162,11 @@ public static void Beep()
             }
         }
 
+        public static void Beep(int frequency, int duration)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
I'm still not sure about this one.  I know that @ellismg was keen on this throwing an exception if it couldn't be implemented properly, and I know that for the niche case where someone's playing music or something, this would matter.  But for the case where someone, for example, is just trying to differentiate a user errors cases with different tones or something like that, just delegating to Beep() would seem like the better choice.  I guess we can start with the PNSE and see if it troubles anyone.